### DISCUSSION TITLE
Setup issue: Could not setup the project, gave type error on "npm start"

### DISCUSSION THREAD

**Discussion by KushalBhurve** [ant-design/ant-design#49423] [Q&A]
I just setup the project on my device and started it up using "npm start" after installing dependancies. But it is giving the following error. I'm kind of a beginner so reaching out for help.

TypeError: (0 , import_token.initInputToken) is not a function

The node version i have is v20.11.1

**Comment by yezhonghu0503** [comment]
use node v16

**Reply by 80666881** [reply]
solve my problem,thanks
